src/DistributionsAD.jl

```diff
@@ -49,14 +43,39 @@ export TuringScalMvNormal,
 
 include("common.jl")
 include("univariate.jl")
+#=
 include("multivariate.jl")
 include("mvcategorical.jl")
 include("matrixvariate.jl")
 include("flatten.jl")
 include("arraydist.jl")
 include("filldist.jl")
-@init @require ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267" begin
-    include("reversediff.jl")
+=#
+function __init__()
+    @require ForwardDiff="f6369f11-7733-5829-9624-2563aa707210" begin
+        using ForwardDiff: @define_binary_dual_op # Needed for `eval`ing diffrules here
```

Comment:
```suggestion
        using .ForwardDiff: @define_binary_dual_op # Needed for `eval`ing diffrules here
```
See my other comment about `.`